reporting_dashboard.tf

```diff
@@ -0,0 +1,73 @@
+locals {
+  reporting_helper_lambda_name = "${local.environment}-dr2-reporting-helper"
+  origin_id_s3_origin          = "s3Origin"
+}
+
+module "dr2_reporting_helper_cloudwatch_event" {
+  source                  = "git::https://github.com/nationalarchives/da-terraform-modules//cloudwatch_events"
+  rule_name               = "${local.environment}-dr2-reporting-helper-schedule"
+  schedule                = "rate(10 minutes)"
+  lambda_event_target_arn = "arn:aws:lambda:eu-west-2:${data.aws_caller_identity.current.account_id}:function:${local.reporting_helper_lambda_name}"
+}
+
+
+module "dr2_reporting_helper_lambda" {
+  source          = "git::https://github.com/nationalarchives/da-terraform-modules//lambda"
+  function_name   = local.reporting_helper_lambda_name
+  handler         = "lambda_function.lambda_handler"
+  timeout_seconds = local.python_timeout_seconds
+  policies = {
+    "${local.reporting_helper_lambda_name}-policy" = templatefile("./templates/iam_policy/reporting_helper_policy.json.tpl", {
+      account_id  = data.aws_caller_identity.current.account_id
+      lambda_name = local.reporting_helper_lambda_name
+      bucket_name = local.reporting_bucket_name
+    })
+  }
+  memory_size = local.python_lambda_memory_size
+  runtime     = local.python_runtime
+
```

Comment:
Could you add the environment variables here? I've added them to [my PR](https://github.com/nationalarchives/dr2-ingest/pull/483/files#diff-930007e96c8ccb8cee7ea72cab45ab3f1a5413e0328eba5b30dedb8c0d378bbbR43)